### ISSUE TITLE
verifyEdgeConsistency failed

### ISSUE THREAD

**Issue by steven-solar** [maickrau/MBG#24]
Running into an issue of MBG crashing when running ribotin on chm13 and specifying verkko nodes for each acro cluster. The first
cluster completes successfully, the next crashes. Wasn't sure if I should place this issue here or in ribotin, but seems to be an MBG issue.

Main log:
```
...
running cluster 1 in folder ./result1
running MBG
MBG command:
/data/solarsj/tools/MBG/bin/MBG -o ./result1/graph.gfa -i ./result1/hifi_reads.fa -k 101 -w 71 -a 2 -u 3 -r 9000 -R 4000 --error-masking=msat --output-sequence-paths ./result1/paths.gaf --only-local-resolve 1> ./result1/mbg_stdout.txt 2> ./result1/mbg_stderr.txt
sh: line 1: 3390703 Aborted                 (core dumped) /data/solarsj/tools/MBG/bin/MBG -o ./result1/graph.gfa -i ./result1/hifi_reads.fa -k 101 -w 71 -a 2 -u 3 -r 9000 -R 4000 --error-masking=msat --output-sequence-paths ./result1/paths.gaf --only-local-resolve > ./result1/mbg_stdout.txt 2> ./result1/mbg_stderr.txt
MBG did not run successfully
run-ribotin.sh: line 20: 3358142 Aborted                 (core dumped) $RIBOTIN/bin/ribotin-verkko -t 32 -i /data/Phillippy2/projects/primate_T2T/steven-temp/coverage-estimation/ribotin/beta_asm --mbg $MBG --graphaligner $GA -c chr13_nodes.txt -c chr14_nodes.txt -c chr15_nodes.txt -c chr21_nodes.txt -c chr22_nodes.txt --orient-by-reference 45S.fa --do-ul
```

`result1/mbg_stderr.txt`:
```
...
try resolve k=4707, replaced 1 nodes with 2 nodes, unitigified 2 nodes to 1 nodes
try resolve k=4754, replaced 0 nodes with 0 nodes
try resolve k=5031, replaced 1 nodes with 4 nodes, unitigified 7 nodes to 1 nodes
try resolve k=5362, replaced 1 nodes with 3 nodes, unitigified 5 nodes to 1 nodes
6670 unitigs after resolving
Building unitig sequences
Reading sequences from ./result1/hifi_reads.fa
MBG: src/MBG.cpp:1208: void verifyEdgeConsistency(const UnitigGraph&, const HashList&, const StringIndex&, const std::vector<CompressedSequence>&, size_t, std::pair<long unsigned int, bool>, std::pair<long unsigned int, bool>): Assertion `unitigSequences[from.first].getExpandedStr(fromIndex, stringIndex) == revCompRaw(unitigSequences[to.first].getExpandedStr(toIndex, stringIndex))' failed.
```

Happy to provide more info/data if needed to investigate further.

**Comment by maickrau** [comment]
Could you also upload `result1/hifi_reads.fa`

**Comment by maickrau** [comment]
Crash fixed in 96813bc
